### ISSUE TITLE
Navigation from field details to database table list fails when tooltip is present on breadcrumb

### ISSUE THREAD

**Issue by lewisl9029** [metabase/metabase#3056]
### Steps to repro:

1. Navigate to Data Reference -> Sample Dataset -> Tables -> People -> Fields -> Created At
2. Hover over Sample Dataset breadcrumb until tooltip appears
![tooltip-breakage](https://cloud.githubusercontent.com/assets/6934200/17195424/6f55face-5412-11e6-99d4-2418e49ae905.png)
3. Click the Sample Dataset breadcrumb

### Expected:

Sample Dataset table list renders correctly

### Actual: 

Sample Dataset table list breaks:
![tooltip-breakage-2](https://cloud.githubusercontent.com/assets/6934200/17195490/cf1221fe-5412-11e6-97c1-661cd04563d7.PNG)

With following error message in console:
```
TypeError: Cannot read property 'componentDidUpdate' of null
    at ReactCompositeComponentWrapper.invokeComponentDidUpdateWithTimer (eval at <anonymous> (app.hot.bundle.js?8be0c37…:1), <anonymous>:65:17)
    at CallbackQueue.notifyAll (eval at <anonymous> (app.hot.bundle.js?8be0c37…:1), <anonymous>:67:22)
    at ReactReconcileTransaction.close (eval at <anonymous> (app.hot.bundle.js?8be0c37…:1), <anonymous>:81:26)
    at ReactReconcileTransaction.closeAll (eval at <anonymous> (app.hot.bundle.js?8be0c37…:1), <anonymous>:204:25)
    at ReactReconcileTransaction.perform (eval at <anonymous> (app.hot.bundle.js?8be0c37…:1), <anonymous>:151:16)
    at ReactUpdatesFlushTransaction.perform (eval at <anonymous> (app.hot.bundle.js?8be0c37…:1), <anonymous>:138:20)
    at ReactUpdatesFlushTransaction.perform (eval at <anonymous> (app.hot.bundle.js?8be0c37…:1), <anonymous>:90:38)
    at flushBatchedUpdates (eval at <anonymous> (app.hot.bundle.js?8be0c37…:1), <anonymous>:173:19)
    at ReactUpdatesFlushTransaction.close (eval at <anonymous> (app.hot.bundle.js?8be0c37…:1), <anonymous>:48:7)
    at ReactUpdatesFlushTransaction.closeAll (eval at <anonymous> (app.hot.bundle.js?8be0c37…:1), <anonymous>:204:25)(anonymous function) @ angular.js?f3fe:10126(anonymous function) @ angular.js?f3fe:7398$broadcast @ angular.js?f3fe:13095afterLocationChange @ angular.js?f3fe:9972(anonymous function) @ angular.js?f3fe:9960$eval @ angular.js?f3fe:12811$digest @ angular.js?f3fe:12623$apply @ angular.js?f3fe:12915(anonymous function) @ angular.js?f3fe:9913(anonymous function) @ angular.js?f3fe:2853forEach @ angular.js?f3fe:325eventHandler @ angular.js?f3fe:2852
```

### Additional info:

Appears to be caused by something in our `Popover` component. No errors observed after replacing `Popover` with `div` in https://github.com/metabase/metabase/blob/master/frontend/src/metabase/components/TooltipPopover.jsx

**Comment by camsaul** [comment]
@lewisl9029 was this fixed by #3053?

**Comment by lewisl9029** [comment]
@camsaul nope. @tlrobinson is working on it and might have a fix soon though.
